docker/apache.conf

```diff
@@ -0,0 +1,14 @@
+# Configure apache
+<VirtualHost *:80>
+    ServerName example.com
```

Comment:
Does this `ServerName` directive need to be changed at all? Would someone need to edit this file in order to properly deploy the image?

If someone wants to deploy this imagine to serve a website listening on "somesite.com", what modifications would they need to make to make it work?